AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAssertImagePreviewActionGroup.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAssertImagePreviewActionGroup">
+        <click selector="{{AdobeStockSection.modal}} {{AdobeStockSection.imagesThumbnails}}" stepKey="clickOnThumbnail"/>
+        <grabAttributeFrom selector="{{AdobeStockSection.imagePreview}}" userInput="src" stepKey="getUrlImageFromCmsPage"/>
+        <click selector="{{AdobeStockSection.savePreview}}" stepKey="clickOnSavePreview"/>
```

Comment:
Please divide this action group to `AdminAdobeStockSavePreview` and `AdminAdobeStockAssertImageSaved`

Reply:
Hi @sivaschenko,

If `AdminAdobeStockAssertImageSaved` is an assertion, I believe it should have "Assert" in the name, so it would be `AdminAssertAdobeStockAssertImageSaved`

Reply:
@hws47a aggree, but then `Assert` appears in the name twice :)